src/js/themes/base.d.ts

```diff
@@ -726,7 +726,11 @@ export interface ThemeType {
     round?: RoundType;
   };
   formField?: {
-    border?: BorderType;
+    border?: BorderType & {
+      error?: {
+        color?: ColorType;
```

Comment:
Is there anything else that would be on border.error? Or it's just color that we can change

Reply:
https://v2.grommet.io/formfield#formField.border.error.color 
Thats it according to the docs

Reply:
also checked code:
https://github.com/grommet/grommet/blob/master/src/js/components/FormField/FormField.js#L264

Reply:
Great thanks!